### ISSUE TITLE
Not parsing `apt search` responses correctly

### ISSUE THREAD

**Issue by 0b1kn00b** [kdeldycke/meta-package-manager#966]
### What happened?

NB, I removed debugging from `mpm --all-managers` call because github is complaining that the comment is too long >65536 characters
with Ubuntu 22.10, xonsh

```shell-session
mpm -v DEBUG --apt search fuse-overlayfs
debug: Verbosity set to DEBUG.
debug: Load configuration matching /home/dayz/.config/mpm/*.{toml,yaml,yml,json,ini,xml}
debug: Pattern is not an URL.
debug: Search local file system.
debug: No configuration file found.
debug: mpm, version 5.12.0
debug: Initial population of user-selected managers: apt
debug: Does <class 'meta_package_manager.managers.apt.APT'> implements Operations.search?
debug: Search for apt in /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/o/var/asdf/plugins/python/shims, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/o/var/asdf/installs/python/3.11.2/bin, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/i/run/gnu, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/o/var/asdf/shims, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/o/rakubrew/bin, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/i/prj/kisslang/bin, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/asdf/bin, /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/i/run, /home/dayz/.nvm/current/bin, /home/dayz/.cargo/bin, /home/dayz/.config/guix/current/bin, /usr/local/sbin, /usr/local/bin, /usr/sbin, /usr/bin, /usr/games, /usr/local/games, /snap/bin, /mnt/dat/prg/haxe, /mnt/dat/bin, /home/dayz/.local/bin, /home/dayz/i/bin
debug: CLI found at /usr/bin/apt
debug: ► /usr/bin/apt --version
debug:   apt 2.4.8 (amd64)
debug: Extracted version: '2.4.8'
debug: Parsed version: <TokenizedString 2.4.8 => (<Token:string='2',integer=2>, <Token:string='4',integer=4>, <Token:string='8',integer=8>)>
debug: apt is deprecated: False; is supported: True; found at: /usr/bin/apt; is executable: True; is fresh: True.
debug: ► /usr/bin/apt --quiet search fuse-overlayfs --names-only
debug:   Sorting...
debug:   Full Text Search...
debug:   fuse-overlayfs/jammy,now 1.7.1-1 amd64 [installed]
debug:     implementation of overlay+shiftfs in FUSE for rootless containers
warning:   WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
0 package total (apt: 0).
```

### Meta Package Manager configuration

```shell-session
╭────────────────────────────┬────────────────────────────────────────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┬──────┬──────────────────┬─────────┬─────────────────────────┬───────────────────────────────────────────────────────┬───────────────────────────────────────────────────────┬─────────────╮
│ ID                         │ Class                                  │ Spec.                                                                                                                                                                                                                                                                                                                                                                                                                           │ Type │ Allowed in conf? │ Exposed │ Env. var.               │ Default                                               │ Value                                                 │ Source      │
├────────────────────────────┼────────────────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┼──────┼──────────────────┼─────────┼─────────────────────────┼───────────────────────────────────────────────────────┼───────────────────────────────────────────────────────┼─────────────┤
│ mpm.all_managers           │ cloup._params.Option                   │ -a, --all-managers                                                                                                                                                                                                                                                                                                                                                                                                              │ bool │                  │ ✓       │ MPM_ALL_MANAGERS        │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.apm                    │ cloup._params.Option                   │ --apm                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_APM                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.apt                    │ cloup._params.Option                   │ --apt                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_APT                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.apt_mint               │ cloup._params.Option                   │ --apt-mint                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_APT_MINT            │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.bar_plugin_path        │ cloup._params.Option                   │ --bar-plugin-path                                                                                                                                                                                                                                                                                                                                                                                                               │ bool │                  │ ✘       │ MPM_BAR_PLUGIN_PATH     │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.brew                   │ cloup._params.Option                   │ --brew                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_BREW                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.cargo                  │ cloup._params.Option                   │ --cargo                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_CARGO               │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.cask                   │ cloup._params.Option                   │ --cask                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_CASK                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.choco                  │ cloup._params.Option                   │ --choco                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_CHOCO               │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.color                  │ click_extra.colorize.ColorOption       │ --color, --ansi / --no-color, --no-ansi                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_COLOR               │ True                                                  │ True                                                  │ DEFAULT     │
│ mpm.composer               │ cloup._params.Option                   │ --composer                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_COMPOSER            │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.config                 │ click_extra.config.ConfigOption        │ -C, --config CONFIG_PATH                                                                                                                                                                                                                                                                                                                                                                                                        │ str  │                  │ ✘       │ MPM_CONFIG              │ /home/dayz/.config/mpm/*.{toml,yaml,yml,json,ini,xml} │ /home/dayz/.config/mpm/*.{toml,yaml,yml,json,ini,xml} │ DEFAULT     │
│ mpm.description            │ cloup._params.Option                   │ --description                                                                                                                                                                                                                                                                                                                                                                                                                   │ bool │                  │ ✓       │ MPM_DESCRIPTION         │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.dnf                    │ cloup._params.Option                   │ --dnf                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_DNF                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.dry_run                │ cloup._params.Option                   │ -d, --dry-run                                                                                                                                                                                                                                                                                                                                                                                                                   │ bool │                  │ ✓       │ MPM_DRY_RUN             │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.emerge                 │ cloup._params.Option                   │ --emerge                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_EMERGE              │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.exclude                │ cloup._params.Option                   │ -e, --exclude [apm|apt|apt-mint|brew|cargo|cask|choco|composer|dnf|emerge|flatpak|gem|mas|npm|opkg|pacaur|pacman|paru|pip|pipx|scoop|snap|steamcmd|vscode|yarn|yay|yum|zypper]                                                                                                                                                                                                                                                  │ list │                  │ ✓       │ MPM_EXCLUDE             │                                                       │                                                       │ DEFAULT     │
│ mpm.flatpak                │ cloup._params.Option                   │ --flatpak                                                                                                                                                                                                                                                                                                                                                                                                                       │ bool │                  │ ✘       │ MPM_FLATPAK             │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.gem                    │ cloup._params.Option                   │ --gem                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_GEM                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.help                   │ click_extra.colorize.HelpOption        │ -h, --help                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_HELP                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.ignore_auto_updates    │ cloup._params.Option                   │ --ignore-auto-updates / --include-auto-updates                                                                                                                                                                                                                                                                                                                                                                                  │ bool │                  │ ✓       │ MPM_IGNORE_AUTO_UPDATES │ True                                                  │ True                                                  │ DEFAULT     │
│ mpm.manager                │ cloup._params.Option                   │ -m, --manager [apm|apt|apt-mint|brew|cargo|cask|choco|composer|dnf|emerge|flatpak|gem|mas|npm|opkg|pacaur|pacman|paru|pip|pipx|scoop|snap|steamcmd|vscode|yarn|yay|yum|zypper]                                                                                                                                                                                                                                                  │ list │                  │ ✓       │ MPM_MANAGER             │                                                       │                                                       │ DEFAULT     │
│ mpm.mas                    │ cloup._params.Option                   │ --mas                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_MAS                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.npm                    │ cloup._params.Option                   │ --npm                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_NPM                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.opkg                   │ cloup._params.Option                   │ --opkg                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_OPKG                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.output_format          │ click_extra.tabulate.TableFormatOption │ -o, --output-format [asciidoc|csv|csv-excel|csv-excel-tab|csv-unix|double_grid|double_outline|fancy_grid|fancy_outline|github|grid|heavy_grid|heavy_outline|html|jira|json|latex|latex_booktabs|latex_longtable|latex_raw|mediawiki|mixed_grid|mixed_outline|moinmoin|orgtbl|outline|pipe|plain|presto|pretty|psql|rounded_grid|rounded_outline|rst|simple|simple_grid|simple_outline|textile|tsv|unsafehtml|vertical|youtrack] │ str  │                  │ ✘       │ MPM_OUTPUT_FORMAT       │ rounded_outline                                       │ rounded_outline                                       │ DEFAULT     │
│ mpm.pacaur                 │ cloup._params.Option                   │ --pacaur                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_PACAUR              │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.pacman                 │ cloup._params.Option                   │ --pacman                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_PACMAN              │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.paru                   │ cloup._params.Option                   │ --paru                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_PARU                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.pip                    │ cloup._params.Option                   │ --pip                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_PIP                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.pipx                   │ cloup._params.Option                   │ --pipx                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_PIPX                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.scoop                  │ cloup._params.Option                   │ --scoop                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_SCOOP               │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.show_params            │ click_extra.config.ShowParamsOption    │ --show-params                                                                                                                                                                                                                                                                                                                                                                                                                   │ bool │                  │ ✘       │ MPM_SHOW_PARAMS         │ False                                                 │ True                                                  │ COMMANDLINE │
│ mpm.snap                   │ cloup._params.Option                   │ --snap                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_SNAP                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.sort_by                │ cloup._params.Option                   │ -s, --sort-by [manager_id|manager_name|package_id|package_name|version]                                                                                                                                                                                                                                                                                                                                                         │ str  │                  │ ✓       │ MPM_SORT_BY             │ manager_id                                            │ manager_id                                            │ DEFAULT     │
│ mpm.stats                  │ cloup._params.Option                   │ --stats / --no-stats                                                                                                                                                                                                                                                                                                                                                                                                            │ bool │                  │ ✓       │ MPM_STATS               │ True                                                  │ True                                                  │ DEFAULT     │
│ mpm.steamcmd               │ cloup._params.Option                   │ --steamcmd                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_STEAMCMD            │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.stop_on_error          │ cloup._params.Option                   │ --stop-on-error / --continue-on-error                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✓       │ MPM_STOP_ON_ERROR       │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.time                   │ click_extra.commands.TimerOption       │ --time / --no-time                                                                                                                                                                                                                                                                                                                                                                                                              │ bool │                  │ ✘       │ MPM_TIME                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.verbosity              │ click_extra.logging.VerbosityOption    │ -v, --verbosity LEVEL                                                                                                                                                                                                                                                                                                                                                                                                           │ str  │                  │ ✘       │ MPM_VERBOSITY           │ INFO                                                  │ INFO                                                  │ DEFAULT     │
│ mpm.version                │ click_extra.version.VersionOption      │ --version                                                                                                                                                                                                                                                                                                                                                                                                                       │ bool │                  │ ✘       │ MPM_VERSION             │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.vscode                 │ cloup._params.Option                   │ --vscode                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_VSCODE              │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.xkcd                   │ cloup._params.Option                   │ -x, --xkcd                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✓       │ MPM_XKCD                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.yarn                   │ cloup._params.Option                   │ --yarn                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_YARN                │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.yay                    │ cloup._params.Option                   │ --yay                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_YAY                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.yum                    │ cloup._params.Option                   │ --yum                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_YUM                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.zypper                 │ cloup._params.Option                   │ --zypper                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_ZYPPER              │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.backup.merge           │ cloup._params.Option                   │ --merge                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✓       │ MPM_MERGE               │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.backup.overwrite       │ cloup._params.Option                   │ --overwrite, --force, --replace                                                                                                                                                                                                                                                                                                                                                                                                 │ bool │                  │ ✓       │ MPM_OVERWRITE           │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.backup.toml_path       │ cloup._params.Argument                 │ [TOML_PATH]                                                                                                                                                                                                                                                                                                                                                                                                                     │ str  │                  │ ✓       │                         │ -                                                     │ -                                                     │ DEFAULT     │
│ mpm.backup.update_version  │ cloup._params.Option                   │ --update-version                                                                                                                                                                                                                                                                                                                                                                                                                │ bool │                  │ ✓       │ MPM_UPDATE_VERSION      │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.install.packages_specs │ cloup._params.Argument                 │ PACKAGES_SPECS...                                                                                                                                                                                                                                                                                                                                                                                                               │ list │                  │ ✓       │                         │                                                       │                                                       │ DEFAULT     │
│ mpm.installed.duplicates   │ cloup._params.Option                   │ -d, --duplicates                                                                                                                                                                                                                                                                                                                                                                                                                │ bool │                  │ ✓       │ MPM_DUPLICATES          │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.outdated.plugin_output │ cloup._params.Option                   │ --plugin-output                                                                                                                                                                                                                                                                                                                                                                                                                 │ bool │                  │ ✓       │ MPM_PLUGIN_OUTPUT       │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.remove.packages_specs  │ cloup._params.Argument                 │ PACKAGES_SPECS...                                                                                                                                                                                                                                                                                                                                                                                                               │ list │                  │ ✓       │                         │                                                       │                                                       │ DEFAULT     │
│ mpm.restore.toml_files     │ cloup._params.Argument                 │ TOML_FILES...                                                                                                                                                                                                                                                                                                                                                                                                                   │ list │                  │ ✓       │                         │                                                       │                                                       │ DEFAULT     │
│ mpm.search.exact           │ cloup._params.Option                   │ --exact / --fuzzy                                                                                                                                                                                                                                                                                                                                                                                                               │ bool │                  │ ✓       │ MPM_EXACT               │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.search.extended        │ cloup._params.Option                   │ --extended / --id-name-only                                                                                                                                                                                                                                                                                                                                                                                                     │ bool │                  │ ✓       │ MPM_EXTENDED            │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.search.query           │ cloup._params.Argument                 │ QUERY                                                                                                                                                                                                                                                                                                                                                                                                                           │ str  │                  │ ✓       │                         │                                                       │                                                       │ DEFAULT     │
│ mpm.search.refilter        │ cloup._params.Option                   │ --refilter / --no-refilter                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✓       │ MPM_REFILTER            │ True                                                  │ True                                                  │ DEFAULT     │
│ mpm.upgrade.all            │ cloup._params.Option                   │ -A, --all                                                                                                                                                                                                                                                                                                                                                                                                                       │ bool │                  │ ✓       │ MPM_ALL                 │ False                                                 │ False                                                 │ DEFAULT     │
│ mpm.upgrade.packages_specs │ cloup._params.Argument                 │ [PACKAGES_SPECS]...                                                                                                                                                                                                                                                                                                                                                                                                             │ list │                  │ ✓       │                         │                                                       │                                                       │ DEFAULT     │
╰────────────────────────────┴────────────────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┴──────┴──────────────────┴─────────┴─────────────────────────┴───────────────────────────────────────────────────────┴───────────────────────────────────────────────────────┴─────────────╯
```


### List package managers

```shell-session
╭────────────┬────────────────────┬────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────┬────────────┬──────────╮
│ Manager ID │ Name               │ Supported      │ CLI                                                                                                    │ Executable │ Version  │
├────────────┼────────────────────┼────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────┼────────────┼──────────┤
│ apm        │ Atom's apm         │ ✓ (deprecated) │ ✘ apm not found                                                                                        │            │          │
│ apt        │ APT                │ ✓              │ ✓ /usr/bin/apt                                                                                         │ ✓          │ ✓ 2.4.8  │
│ apt-mint   │ Linux Mint's apt   │ ✓              │ ✓ /usr/bin/apt                                                                                         │ ✓          │ ✘        │
│ brew       │ Homebrew Formulae  │ ✓              │ ✘ brew not found                                                                                       │            │          │
│ cargo      │ Rust's cargo       │ ✓              │ ✓ /home/dayz/.cargo/bin/cargo                                                                          │ ✓          │ ✓ 1.54.0 │
│ cask       │ Homebrew Cask      │ ✘ macOS only   │ ✘ brew not found                                                                                       │            │          │
│ choco      │ Chocolatey         │ ✘ Windows only │ ✘ choco not found                                                                                      │            │          │
│ composer   │ PHP's Composer     │ ✓              │ ✘ composer not found                                                                                   │            │          │
│ dnf        │ DNF                │ ✓              │ ✘ dnf not found                                                                                        │            │          │
│ emerge     │ Emerge             │ ✓              │ ✘ emerge not found                                                                                     │            │          │
│ flatpak    │ Flatpak            │ ✓              │ ✓ /usr/bin/flatpak                                                                                     │ ✓          │ ✓ 1.12.7 │
│ gem        │ Ruby Gems          │ ✓              │ ✓ /usr/bin/gem                                                                                         │ ✓          │ ✓ 3.3.5  │
│ mas        │ Mac AppStore       │ ✘ macOS only   │ ✘ mas not found                                                                                        │            │          │
│ npm        │ Node's npm         │ ✓              │ ✓ /home/dayz/.nvm/current/bin/npm                                                                      │ ✓          │ ✓ 9.1.3  │
│ opkg       │ OPKG               │ ✓              │ ✘ opkg not found                                                                                       │            │          │
│ pacaur     │ Pacaur             │ ✓              │ ✘ pacaur not found                                                                                     │            │          │
│ pacman     │ Pacman             │ ✓              │ ✘ pacman not found                                                                                     │            │          │
│ paru       │ Paru               │ ✓              │ ✘ paru not found                                                                                       │            │          │
│ pip        │ Pip                │ ✓              │ ✓ /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/o/var/asdf/installs/python/3.11.2/bin/python3 │ ✓          │ ✓ 22.3.1 │
│ pipx       │ Pipx               │ ✓              │ ✓ /mnt/dat/i/prj/haxe/published/stx_workspace/.local/stx/o/var/asdf/installs/python/3.11.2/bin/pipx    │ ✓          │ ✓ 1.1.0  │
│ scoop      │ Scoop              │ ✘ Windows only │ ✘ scoop not found                                                                                      │            │          │
│ snap       │ Snap               │ ✓              │ ✓ /usr/bin/snap                                                                                        │ ✓          │ ✓ 2.58.2 │
│ steamcmd   │ Valve Steam        │ ✓              │ ✘ steamcmd not found                                                                                   │            │          │
│ vscode     │ Visual Studio Code │ ✓              │ ✓ /usr/bin/code                                                                                        │ ✓          │ ✓ 1.73.0 │
│ yarn       │ Node's yarn        │ ✓              │ ✘ yarn not found                                                                                       │            │          │
│ yay        │ Yay                │ ✓              │ ✘ yay not found                                                                                        │            │          │
│ yum        │ YUM                │ ✓              │ ✘ yum not found                                                                                        │            │          │
│ zypper     │ Zypper             │ ✓              │ ✘ zypper not found                                                                                     │            │          │
╰────────────┴────────────────────┴────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────┴────────────┴──────────╯
```


### Meta Package Manager version

```shell-session
mpm, version 5.12.0
```

**Comment by kdeldycke** [comment]
There was an issue in the regex parsing `apt`'s results. I just fixed it upstream in 8bd5a770a5171b5d78fac7b771b64ce2c7dc4145.

The fix will be available in the upcoming release of `mpm` 5.13.0.

**Comment by kdeldycke** [comment]
FYI meta-package-manager v5.13.0 has just been released.
